### PR TITLE
Fantomstarters and more

### PR DESCRIPTION
**Pull request by Rashmi-278**
Add:

Fantomstarter
BaconCoin
Wagumi
Unslashed

### CODE REVIEW

File: protocols/baconcoin/overview.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
+# BaconCoin
+
+## Overview
```
Comment (cryptosabago) [inline protocols/baconcoin/overview.md:3]: Add a new line please.

File: protocols/baconcoin/overview.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+### Why Bacon?
```
Comment (cryptosabago) [inline protocols/baconcoin/overview.md:4]: Please see prev. comment

File: protocols/baconcoin/overview.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+Almost everyone has heard the phrase “Bring home the bacon,” which roughly means to “earn the money that is needed to live.” Purchasing bHOME allows anyone with a crypto wallet to access the same asset class used by banks and governments to preserve and grow their wealth.
+
+### Intro
```
Comment (cryptosabago) [inline protocols/baconcoin/overview.md:7]: Please see prev. comment

File: protocols/fantomstarter/overview.md [hunk 4]
```diff
@@ -0,0 +1,54 @@
+# FantomStarter
+
+## Overview
+
+FantomStarter DAO empowers the token and NFT holders to decide about the next projects to be launched on the platform.
+
+### Why FantomChain ?
```
Comment (cryptosabago) [inline protocols/fantomstarter/overview.md:7]: Please see prev. comment.

File: protocols/fantomstarter/overview.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+sets Fantom apart and why we opted to create the smart contract for the FS launchpad on
+Fantom Opera.
+
+In doing so, it has opened up investing opportunities on all the major chains, not just one.
+The FantomStarter launchpad provides logistics on more than one chain. It supports
+Fantom (FTM), Binance Smart Chain (BSC), Ethereum (ETH), and Polygon (Matic) and many
+others. This key differentiator allows Founders the freedom to focus on what they do best;
+build their project.
+
+## Governance
```
Comment (cryptosabago) [inline protocols/fantomstarter/overview.md:25]: Please see prev. comment.

File: protocols/fantomstarter/overview.md [hunk 6]
```diff
@@ -0,0 +1,54 @@
...
+overlooked by the FS team but well-liked by the community can
+be picked up through governance as Token & NFT holders use
+their voting power to champion the projects they believe in.
+Governance requires the voting power of the community, which
+further expands the use-case of our $FS Tokens & NFTs. This
+built-in redundancy sets in motion the execution of community
+led projects and makes sure every opportunity for success is
+given.
+
+### Snapshot
```
Comment (cryptosabago) [inline protocols/fantomstarter/overview.md:38]: same here

File: protocols/fantomstarter/overview.md [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+led projects and makes sure every opportunity for success is
+given.
+
+### Snapshot 
+FantomStarter will empower the community by letting them vote on future IDO listings. Powered by Snapshot; a decentralized voting system that provides flexible governance across most blockchains, the FantomStarter community will now act as an additional crowd-sourced layer of assurance making sure the projects listed are not only internally audited by an experienced; dedicated team, but also reflective of variables outside of our control.
+Whilst we utilize our expertise on the business, legal and compliance side, the FantomStarter community will use their collective expertise of market demand to determine if a project has what it takes to succeed beyond our initial recommendations.
+
+Every Smart subscriber can vote based on how much tokens he used for the Smart subscription. Governance is fully on chain and to create a proposal you need to be Tier 5 or above. Both users subscribed with FS tokens but also FS Key NFT holders ca vote and propose projects. Every proposal has a 24hr cooldown and a 3 days duration.
+
+## Join Us
```
Comment (cryptosabago) [inline protocols/fantomstarter/overview.md:44]: same here

File: protocols/fantomstarter/overview.md [hunk 8]
```diff
@@ -0,0 +1,54 @@
...
+Contact us on [Telegram](https://t.me/fantomstarter) or [Discord](https://discord.com/invite/nsMxVcNwy6)
+
+## Additional links
```
Comment (cryptosabago) [inline protocols/fantomstarter/overview.md:47]: here as well

**Comment by cryptosabago** [comment]
There is a minor formatting issue on some of these files.

File: protocols/baconcoin/overview.md (hunk 2 above)
Comment (Rashmi-278) [reply protocols/baconcoin/overview.md:4]: Please see prev. comment

File: protocols/baconcoin/overview.md (hunk 3 above)
Comment (Rashmi-278) [reply protocols/baconcoin/overview.md:7]: Please see prev. comment

File: protocols/fantomstarter/overview.md (hunk 4 above)
Comment (Rashmi-278) [reply protocols/fantomstarter/overview.md:7]: Please see prev. comment

File: protocols/fantomstarter/overview.md (hunk 5 above)
Comment (Rashmi-278) [reply protocols/fantomstarter/overview.md:25]: Please see prev. comment

**Comment by Rashmi-278** [comment]
baconcoin
fatomstarter
unslashed
image

Note: The broken images will show up once they are on production and protocol-info PRs are merged

**Review by CarlGocht — APPROVED** [review]

**Pull request merged**